pkg/codegen/docs/gen.go

```diff
@@ -137,15 +137,16 @@ type header struct {
 
 // property represents an input or an output property.
 type property struct {
+	ID string
 	// DisplayName is the property name with word-breaks.
 	DisplayName        string
 	Name               string
 	Comment            string
 	Type               propertyType
 	DeprecationMessage string
-
-	IsRequired bool
-	IsInput    bool
+	Link               string
+	IsRequired         bool
```

Comment:
Not a big deal but what do you think about restoring the new-line character before this property to separate the `bool`s?